book/networking/p2p/network/README.md

```diff
@@ -0,0 +1,69 @@
+# Network
+
+TODO: Some high-level overview of peer-to-peer networking, either here or in a short chapter one level up in the tree for `p2p`
+
+The `network` crate is responsible for managing the node's connection to the Ethereum peer-to-peer (P2P) network, enabling communication with other nodes via the [various P2P subprotocols](https://github.com/ethereum/devp2p).
+
+TODO: Link to the "bird's eye docs" in `crates/net/network/src/lib.rs`?
+
+Reth's P2P networking consists primarily of 4 ongoing tasks:
+- **Discovery**: Discovers new peers in the network
+- **Transactions**: Accepts, requests, and broadcasts mempool transactions
+- **ETH Requests**: Responds to incoming requests for headers and bodies
+    - TODO: Doesn't also issue requests for these?
+    - TODO: Are headers and bodies all that it's concerned with?
```

Comment:
For now, there's also `GetNodeData` which was deprecated in v67 and `GetReceipts` which isn't very useful, at least in reth.